### DISCUSSION TITLE
devtools를 이용한 live server 구축

### DISCUSSION THREAD

**Discussion by myrhymetree** [Convergence-Project/step-service#50] [Ideas]
## live server 활성화
프론트엔드 개발을 하다보면 타임리프로 작성된 코드와 맞춰 보려고, 서버를 계속 재시작 한 경험이 있을것이다.
그런 귀찮음과 시간 소요를 줄이기 위해서 spring-devtools에서는 리소스 변경이 있을 시 자동으로 컴파일 해주는 기능이 있다.

Gradle
```
developmentOnly 'org.springframework.boot:spring-boot-devtools'
```
<img width="285" alt="image" src="https://github.com/Convergence-Project/step-backend/assets/94158097/59e1612a-712a-44e2-83c4-4684c65eb26d">
<br/> '구성 편집' 클릭 -> 
<img width="781" alt="image" src="https://github.com/Convergence-Project/step-backend/assets/94158097/c22416fd-e468-4467-a17e-9078727130ef">
<br/> '옵션 수정' 클릭 -> <br>
<img width="500" alt="image" src="https://github.com/Convergence-Project/step-backend/assets/94158097/f4746f7c-84d8-4ddc-a245-e2c849a822f9">
<br/> '프레임 비활성화 시' 클릭 -> '리소스 업데이트' 클릭
<br/><br/>
설정하고 나면 변경사항이 있을 경우 페이지 새로고침하면 적용됨.
<br/><br/>
<br/>

[참고](https://velog.io/@bread_dd/Spring-Boot-Devtools)

**Comment by bbbbooo** [comment]
좋아요 좋아요
